src/ontology/cl-edit.owl

```diff
@@ -31101,6 +31122,99 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36543915") Annotation(ob
 AnnotationAssertion(rdfs:label obo:CL_4033026 "lung perichondrial fibroblast")
 EquivalentClasses(obo:CL_4033026 ObjectIntersectionOf(obo:CL_4033025 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002048)))
 
+# Class: obo:CL_4033027 (diffuse bipolar 1 cell)
```

Comment:
Of note, In the two references this cell type has been written as:

"diffuse bipolar cell type DB1"
"DB1 bipolar cell"
"DB1 cell" (currently a related synonym)

The label is likely the expanded form of "DB1 cell".

Reply:
the two first terms look like a wrong usage of the abbreviation, as it reads as `diffuse bipolar cell type diffuse bipolar 1` and `diffuse bipolar 1 bipolar cell`. I would like to avoid adding these half abbreviations in the ontology.